src/Query.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\DBAL;
+
+use Doctrine\DBAL\Types\Type;
+
+/**
+ * A SQL query together with its bound parameters.
+ *
+ * @psalm-immutable
+ */
+class Query
```

Comment:
Not sure if you expected `Query` in the `Doctrine\DBAL` namespace?

Reply:
Ah, no, I expected it under `Doctrine\DBAL\Exception`

Reply:
@morozov I can see that you resolved this, do you want to keep `Query` in the `Doctrine\DBAL` namespace after all?